crates/common/src/test_utils.rs

```diff
@@ -1,12 +1,18 @@
 use crate::{
     hashchain::Hashchain,
-    operation::{Operation, PublicKey, ServiceChallenge},
+    operation::{Operation, ServiceChallenge, SigningKey, VerifyingKey},
     tree::{InsertProof, KeyDirectoryTree, Proof, SnarkableTree, UpdateProof},
 };
 use anyhow::{anyhow, Result};
-use ed25519_dalek::{SigningKey, VerifyingKey};
+#[cfg(not(feature = "secp256k1"))]
+use ed25519_dalek::SigningKey as Ed25519SigningKey;
 use jmt::{mock::MockTreeStore, KeyHash};
-use rand::{rngs::StdRng, Rng};
+use rand::{
+    rngs::{OsRng, StdRng},
+    Rng,
+};
+#[cfg(feature = "secp256k1")]
```

Comment:
Please open an issue that we should have test_utils run the tests for both variants always, instead of via a feature